docs/best_practice/crate_ami.txt

```diff
@@ -42,20 +46,41 @@ Launch the AMI
 You can launch one or more instances with the Crate AMI using the ``run-instances``
 command utilizing the ``--image-id`` command line option. Use the
 available parameters of the command to configure its start-up behavior and
-hardware configuration. A valid key-pair is necessary to connect to the instances 
+hardware configuration. A valid key-pair is necessary to connect to the instances
 later on.
 
 .. code-block:: bash
 
     $ aws ec2 run-instances --image-id <AMI-ID> --count <NR-OF-INSTANCES> --instance-type <INSTANCE-TYPE> --user-data <USER-DATA>  --key-name <KEY-NAME>
 
+.. _cloud_init:
+
+Cloud-Init
+----------
+
+Crate AWS AMI uses `Cloud-Init`_ to setup all necessary configurations on
```

Comment:
the ami creates a basic crate configuration at first launch (using cloud-init). this includes: